site/themes/contour/layouts/partials/getting-involved.html

```diff
@@ -0,0 +1,17 @@
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+<div>
+    <section class="jumbotron text-center " style="background-color:#24c7d3;color:#fff; padding: 50px;">
+        <div class="container">
+            <h1>Get involved</h1>
+            <p class="lead" style="color:#fff;">Join the mailing list to get invites to meetings.</p>
+            <p style="margin-top: 30px;"><a href="https://groups.google.com/g/project-contour" target="_blank"><button
+                        type="button" class="btn btn-primary btn-lg">Join the mailing list</button></a></p>
```

Comment:
we would strongly prefer not to bring in another framework/CSS etc. but use what the site has already

we can use a button similar to this one rather than one from bootstrap: https://github.com/projectcontour/contour/blob/ccb79f8688b9286bfc38217f7029524be2de99e7/site/themes/contour/layouts/partials/hero.html#L7

you can try to use a structure similar to the linked one for this layout